awx/main/scheduler/task_manager.py

```diff
@@ -87,17 +87,32 @@ def job_blocked_by(self, task):
         return None
 
     def get_tasks(self, status_list=('pending', 'waiting', 'running')):
-        jobs = [j for j in Job.objects.filter(status__in=status_list).prefetch_related('instance_group')]
-        inventory_updates_qs = (
-            InventoryUpdate.objects.filter(status__in=status_list).exclude(source='file').prefetch_related('inventory_source', 'instance_group')
-        )
-        inventory_updates = [i for i in inventory_updates_qs]
-        # Notice the job_type='check': we want to prevent implicit project updates from blocking our jobs.
-        project_updates = [p for p in ProjectUpdate.objects.filter(status__in=status_list, job_type='check').prefetch_related('instance_group')]
-        system_jobs = [s for s in SystemJob.objects.filter(status__in=status_list).prefetch_related('instance_group')]
-        ad_hoc_commands = [a for a in AdHocCommand.objects.filter(status__in=status_list).prefetch_related('instance_group')]
-        workflow_jobs = [w for w in WorkflowJob.objects.filter(status__in=status_list)]
-        all_tasks = sorted(jobs + project_updates + inventory_updates + system_jobs + ad_hoc_commands + workflow_jobs, key=lambda task: task.created)
+        all_tasks = []
```

Comment:
`all_tasks` becomes misleading as a variable name now.

While I don't want to dive off the abyss of refactors, this does call into question what all the other task manager methods are expecting, like the suspicious `get_inventory_source_tasks`. Should we maybe categorize which methods require what statuses? Maybe. I don't feel totally comfortable with this merge until I have at least some cursory understanding of that.